pyproject.toml

```
@@ -16,3 +16,151 @@ multi_line_output = 3
 include_trailing_comma = true
 use_parentheses = true
 skip = "migrations"
+
+[tool.poetry]
+name = "redash"
+version = "23.09.0-dev"
+description = "Make Your Company Data Driven. Connect to any data source, easily visualize, dashboard and share your data."
+authors = ["Your Name <you@example.com>"]
```

```suggestion
authors = ["Redash maintainers and contributors"]
```

Or does it have to be an email address?